nodes/sample_host.json

```diff
@@ -1,5 +1,5 @@
 {
-  "run_list":["role[mysql]","role[rails_passenger]", "role[sysadmins]"],
+  "run_list":["role[mysql]","role[remove_applications]", "role[rails_passenger]", "role[sysadmins]"],
```

Comment:
Do we want to list this in the default sample? This kind of feels like an expert feature :)